extensions/amp-date-countdown/0.1/amp-date-countdown.js

```diff
@@ -209,15 +217,17 @@ export class AmpDateCountdown extends AMP.BaseElement {
 
     if (this.endDate_) {
       epoch = Date.parse(this.endDate_);
+    } else if (this.timeleftMs_) {
+      epoch = Date.parse(new Date()) + this.timeleftMs_;
```

Comment:
Last change I see: `Date.parse` is unneeded here. It should be `Number(new Date()) + this.timeleftMs_`